src/components/Home.js

```diff
@@ -2,24 +2,21 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { Redirect } from 'react-router'
 
-
-
 export const Home = ({ ticket }) => {
-    if (ticket && ticket.originId) {
-        return <Redirect to={`/tickets/${ticket.originId}`} />
-    }
-    return <Redirect to={'/search'} />
+  if (ticket && ticket.originId) {
+    return <Redirect to={`/tickets/${ticket.originId}`} />
+  }
+  return <Redirect to={'/search'} />
 }
 
 export const mapStateToProps = (state) => {
     // this gross thing finds all actual tickets, omitting the refresh metadata, and then picks the most recently visited one out
     // the sort function works because the dates are UTC and thus lexically sortable to start with
-    return {
-        ticket: (state.tickets && state.tickets.map) ? Object.values(state.tickets.map).filter(ele => { return ele.lastRefresh })
+  return {
+    ticket: (state.tickets && state.tickets.map) ? Object.values(state.tickets.map).filter(ele => { return ele.lastRefresh })
```

Comment:
This makes me wonder what sort of rules we could add related to use of the ternary operator. I like

`condition ? doIfTrue() : doIfFalse()`

iff it all fits on one line, but if it has to be split into multiple lines, I usually find

`condition`

`  ? doIfTrue()`

`  : doIfFalse()`

if it's too long for one line. What do you think?

Reply:
Agreed. Will look into a lint rule for this.